doc/specs/OOBE-Dialog.md

```diff
@@ -0,0 +1,169 @@
+
+### 2.2. Expected Impact: Customer, and Technology Outcomes
+
+- **High Reliability:** Less than 0.1% crash rate.
+- **Increased Activation:** 30% or more increase in usage in all PowerToys features.
+- **High User Retention:** 25% or more active PowerToys users after 28 days.
+- **High OOBE Completion Rate:** 50%  or more of New PowerToys users reach the last page of the OOBE prompt.
+
+## 3. Requirements
+
+The base of this OOBE dialog was attributed by [Niel’s mock-up](https://github.com/microsoft/PowerToys/issues/1285).
+
+### 3.1. Functional Requirements
+
+#### 3.1.1. Functional Requirements Overview
+
+|No. | Requirement | Pri |
+| - | - | - |
+|1 | The OOBE Dialog window should launch immediately after installation. | P0 |
+|2 | The OOBE Dialog should consist of a collection of pages that describe the available PowerToys. | P0 |
+|3 | On launch, present the Welcome/Landing page in the OOBE Dialog window based on the template described in [Figure 5.1.1.](#511-oobe-dialog-layout) | P1 |
+|4 | Traverse the various OOBE Dialog pages via an list view options on the left-hand pane. | P0 |
+|5 | Alternatively traverse the various OOBE Dialog pages via an on-screen next/previous buttons on each page. | P1 |
+|6 | At any time, the user should be able to exit the OOBE dialog by closing the window. | P0 |
+|7 | Alternatively, the user should be able to exit the OOBE dialog by pressing the ESC key. | P1 |
+|8 | When the OOBE dialog is exited, save all the options that were set by the user. | P0 |
+|9 | In the PowerToys settings “General” menu, add a button to re-launch the OOBE dialog under the “Run at startup” option. | P0 |
+|10 | The OOBE dialog window should be able to be enlarged to accommodate accessibility needs. | P0 |
+|11 | The OOBE dialog should have a preset minimum window size to ensure all content can be viable with the need to scroll the page. | P1 |
+
+#### 3.1.2. Page Layout
+
+| No. | Requirement | Pri |
+| - | - | - |
+|1 | Each page of the OOBE dialog should focus on the details and settings of a single PowerToy. | P0 |
+|2 | A page should include textual information that gives an overview of its PowerToy's basic functions. | P0 |
+|3 | A page should include a hero image/GIF that provides a visual representation of its PowerToy's core functionality. | P0 |
+|4 | A page should include a link to that PowerToys overview page on Microsoft Docs. | P0 |
+|5 | For applicable PowerToys, include a &quot;Launch&quot; button on the page to allow users to instantiate the PowerToy directly from the OOBE dialog window. | P1 |
```

Comment:
We should reconsider this, only some modules can be launched, other modules requires to first open Explorer and select some files, or start moving a window etc.
This makes the various modules first experience inconsistent and also doesn't teach the user on how to activate a module, wouldn't be better to provide the real description on how to use the module? For modules that can launched directly, wouldn't be better to put the actual shortcut to use?

Reply:
@enricogior Yeah that might make more sense to have a 'shortcut visual' that shows the shortcut.

- Can we load in the saved shortcut from Settings (in case the user updates the shortcut, and opens the OOBE later on) instead of hardcoding the default shortcut?
- I'm assuming if the user would have OOBE open and use the shortcut that the module will open?

![image](https://user-images.githubusercontent.com/9866362/105478518-90946100-5ca3-11eb-994a-3320f7ee028e.png)

Reply:
@enricogior That's understandable. We added this based on the user research feedback that it wasn't clear, even with shortcuts displayed, that users could actually (and are encouraged to) try out the various PowerToys while clicking through the OOBE dialog. Having the launcher button makes it explicit that this is something users should try out, but I agree that the preferred interaction is through using the shortcuts. I'm okay taking off the launch button, but we should figure out how to place more emphasis on interacting with PowerToys throughout the OOBE dialog via the shortcuts so that it's clear to end-users that they should do this.

> * I'm assuming if the user would have OOBE open and use the shortcut that the module will open?

@niels9001 yes this is the intended behavior. I like the visual short-cut, but I'm not sure if it adds the needed emphasis that users should "try it out", so I'm inclined to just leave the shortcut in the same text box that's displayed in settings, unless there's evidence that proves otherwise.